DMX.Core.Api/Models/Configurations/ApiConfigurations.cs

```diff
@@ -6,7 +6,8 @@ namespace DMX.Core.Api.Models.Configurations
 {
     public class ApiConfigurations
     {
-        public string Url { get; set; }
-        public string AccessKey { get; set; }
+        public string Url { get; set; }
+        public string GetAllDevicesAccessKey { get; internal set; }
+        public string GetAvailableDevicesAccessKey { get; internal set; }

```

Comment:
Remove the `internal` part. we don't need it.
Also properties should not be named as actions. Actios/Methods has verbs, properties have nouns.

For instance, GetStudentName() that's a method/action but StudentName is a property